erdjs/src/smartcontracts/query.ts

```diff
@@ -120,6 +122,7 @@ export class ContractReturnData {
         this.asBuffer = Buffer.from(asBase64, "base64");
         this.asHex = this.asBuffer.toString("hex");
         this.asNumber = parseInt(this.asHex, 16) || 0;
+        this.asBool = this.asNumber == 1 ? true : false;
```

Comment:
Not-yet-established convention for Smart Contract results, though I've changed to `this.asNumber != 0` (since we are actually more used to that).

In a future PR, we will change `ContractReturnData` to `// TODO: use types & codecs`.